<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_comprar</class>
 <widget class="QDialog" name="tela_comprar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_dados_compra">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>790</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Dados da compra</string>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_dados_fornecedor">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>790</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Dados do Fornecedor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="lb_codigo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>15</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>9</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_razao_social">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>15</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Razão Social:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_razao_social">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>15</y>
      <width>588</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nome">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>40</y>
      <width>588</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>34</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cnpj">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>40</y>
      <width>32</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CNPJ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cnpj">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>40</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>00.000.000/0000-00; </string>
    </property>
    <property name="text">
     <string>../-</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
